operator-common/src/main/java/io/strimzi/operator/common/Operator.java

```diff
@@ -81,6 +86,7 @@ default void reconcileThese(String trigger, Set<NamespaceAndName> desiredNames,
             CompositeFuture.join(futures).map((Void) null).onComplete(handler);
         } else {
             resourceCounter(namespace).set(0);
+            pausedResourceCounter(namespace).set(0);
```

Comment:
when running cluster wide operator, this would create metrics with `namespace=*`. I do not think we want that. I guess the question is whether we want to emit the metrics of 0 resources (since I guess no metric would also count as 0). And if we do, we should again if-it to set this only when it doesn't watch all namespaces.

Reply:
Actually, these two lines 88+89 are not needed at all, or? They just set the 0 again already set on 72-77.